lib/beaker/host/windows/exec.rb

```diff
@@ -119,6 +119,30 @@ def selinux_enabled?()
     false
   end
 
+  # Create the provided directory structure on the host
+  # @param [String] dir The directory structure to create on the host
+  # @return [Boolean] True, if directory construction succeeded, otherwise False
+  def mkdir_p dir
+    cmd = "mkdir -p '#{dir}'"
```

Comment:
This is inconsistent with the Unix behavior due to us reverting the single quoting change that was made previously since it broke interpolated characters in the path such as `~`, which has widespread usage. If you want to prevent globbing, you may be able to use double quotes, instead?

Reply:
Thanks for your suggestion, I tested out and works great!

Reply:
Could you add the double quoting on Unix hosts so that this behavior is consistent between platforms? I can imagine it being pretty frustrating if the behavior varies. 😄

Reply:
Updated the unix methods so windows with cygwin will inherit them.